community/cypher/cypher/src/test/scala/org/neo4j/cypher/EagerizationAcceptanceTest.scala

```diff
@@ -432,8 +580,7 @@ class EagerizationAcceptanceTest extends ExecutionEngineFunSuite with TableDrive
 
     val query = "MATCH (a:Person) OPTIONAL MATCH (a)-[r1]-() DELETE a, r1"
 
-    assertNumberOfEagerness(query, 0)
-    assertStats(updateWithBothPlanners(query), nodesDeleted = 2, relationshipsDeleted = 6)
```

Comment:
Why was this line removed? I think we added it back when we cherry-picked into the other PR, @pontusmelke, but still it's strange.